packages/deployer/test/sample-project/test/helpers/initializer.js

```diff
@@ -22,19 +22,7 @@ function bootstrap() {
 
 async function deploySystemIfNeeded() {
   if (!proxyAddress) {
-    const deployment = getDeployment();
```

Comment:
Had to remove this:

The idea of this was to avoid local deployments every time tests are ran. Worked nicely when just changing test files, but it doesn't detect when contract changes and re-deploy the system.

Without this, every time tests are run, the system will be re-deployed. It's fast enough for now anyway.